go/kbfs/libfs/tlf.go

```diff
@@ -55,7 +58,24 @@ func FilterTLFEarlyExitError(ctx context.Context, err error, log logger.Logger,
 			name)
 		return true, nil
 
+	case libkbfs.NonExistentTeamForHandleError:
```

Comment:
Why not?
```case kbfsmd.ServerErrorWriteAccess, libkbfs.NonExistentTeamForHandleError:```

Reply:
Good point, I always forget you can do that with `(type)` switches as long as you don't actually use the typed variable.  Magic!